src/native/components/Noticias.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { StyleSheet, FlatList, TouchableOpacity, RefreshControl, Image, View } from 'react-native';
+import { Container, Content, Card, CardItem, Body, Text, Button } from 'native-base';
+import { Actions } from 'react-native-router-flux';
+import getRNDraftJSBlocks from 'react-native-draftjs-render';
+import { distanceInWordsToNow } from 'date-fns';
```

Comment:
any particular reason to use this over moment?

Reply:
The repo for `date-fns` clearly describes it as a replacement for moment?

Reply:
i like this better cuz has better syntax and uses pure functions unlike moment were u have to create objects and stuff